src/OpenTelemetry.Instrumentation.AWSLambda/README.md

```diff
@@ -15,16 +15,54 @@ Add `AddAWSLambdaConfigurations()` to `TracerProvider`.
 ```csharp
 TracerProvider tracerProvider = Sdk.CreateTracerProviderBuilder()
         // add other instrumentations
-        .AddAWSLambdaConfigurations()
+        .AddAWSLambdaConfigurations(options => options.DisableAwsXRayContextExtraction = true)
         .Build();
 ```
 
+### AWSLambdaInstrumentationOptions
+
+`AWSLambdaInstrumentationOptions` contains various properties to configure
+AWS lambda instrumentation.
+
+#### `DisableAwsXRayContextExtraction` (optional)
```

Comment:
"(optional)" can be omitted, as all options are optional and this is as expected I think.

Reply:
Removed.